docs/reference/providers/faas-providers.md

```diff
@@ -0,0 +1,25 @@
+# `faas-provider` Interface
+The `faas-provider` provides the CRUD API for Functions as well as the invoke capabilities.
+
+The OpenFaaS team builds and uses the [`faas-provider`](https://github.com/openfaas/faas-provider) project to define the core interface and standard utilities of an OpenFaaS provider.  This interface, written in Go, ensures that you can create a provider which is compliant and compatible with the OpenFaaS gateway.
+
+
+## Official Providers
+
+### faas-memory
+[`faas-memory`](https://github.com/openfaas-incubator/faas-memory) uses in-memory objects to store state and is provided as an official example implementation and for testing purposes.
+
+### faas-netes
```

Comment:
Could you change the heading to Kubernetes providers (then we can add the operator) and Swarm provider?

Reply:
Done